docs/glsl-fuzz-walkthrough.md

```diff
@@ -23,13 +23,13 @@ in this walkthrough.
 
 ## Requirements
 
-**Summary:** the latest release zip and worker applications, Java 8+, and Python 3.5+.
+**Summary:** the latest release zip and worker applications, Java 8+, Python 3.5+, and `adb` on your path.
```

Comment:
Possibly make it clear that adb is optional if you don't care about Android, e.g.:

"... Python 3.5+, and (if you want to do Android testing) `adb` on your path."

Reply:
Good point. Done.